### DISCUSSION TITLE
My bows

### DISCUSSION THREAD

**Discussion by heavyutils** [geode-sdk/sdk#844] [General]
Why does it take longer to update to 2.206 than actual 2.2? The latter seems to be much bigger.

**Comment by iNUCi** [comment]
no it didn't. the first geode version for 2.2 only came out like 1 month later

**Comment by prevter** [comment]
Because 2.206 brought the 64 bit upgrade. It basically means that modding has to start from scratch, because of a completely new architecture.
Also macOS now supports both Intel and M1 chips, which means there's twice the work to do as well.
